.jules/runs/0123d60d-1355-4342-beef-d604f04d95d2/pr_body.md

```
@@ -0,0 +1,22 @@
+# PR Glass Cockpit
+
+## Type
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)